kitsune/settings.py

```diff
@@ -411,6 +411,7 @@ def parse_conn_max_age(value):
 STATICFILES_DIRS = (
     path("assets"),  # emulate bower and css cache busting
     path("jsi18n"),  # Collect jsi18n so that it is cache-busted
+    path("dist"),
```

Comment:
super nit: shall we use something different as a name here? Or even use the `static` folder as the webpack output? Something like static/webpack or similar